app/src/main/java/org/dhis2/usescases/main/program/ProgramAnimation.kt

```diff
@@ -0,0 +1,39 @@
+package org.dhis2.usescases.main.program
+
+import android.animation.ValueAnimator
+import android.graphics.drawable.GradientDrawable
+import android.view.animation.OvershootInterpolator
+import org.dhis2.Bindings.dp
+import org.dhis2.BuildConfig
+
+class ProgramAnimation {
+
+    fun initBackdropCorners(gd: GradientDrawable) {
+        if (BuildConfig.FLAVOR != "dhisUITesting") {
+            ValueAnimator.ofInt(0, 16.dp)
+                .apply {
+                    duration = 700
+                    startDelay = 700
+                    interpolator = OvershootInterpolator()
+                    addUpdateListener {
+                        val value = (it.animatedValue as Int).toFloat()
+                        gd.cornerRadii = floatArrayOf(value, value, value, value, 0f, 0f, 0f, 0f)
+                    }
+                }.start()
```

Comment:
PropertyValuesHolder scalex = PropertyValuesHolder.ofFloat();
            animation = ObjectAnimator.ofPropertyValuesHolder(binding.enrollmentButton, scalex, scaley);
            animation.setDuration(700);
            animation.setStartDelay(700);
            animation.setInterpolator(new OvershootInterpolator());
            animation.addUpdateListener(new ValueAnimator.AnimatorUpdateListener() {
                @Override
                public void onAnimationUpdate(ValueAnimator animation) {
                    
                }
            });
            start

Reply:
I think is straight :+1: .